Webdev4.html

```diff
@@ -0,0 +1,163 @@
+
+    <script>
+            var map = L.map('map').setView([46.79069, -92.083366], 11);
+
+            L.tileLayer('https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png', {
+                attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors &copy; <a href="https://carto.com/attributions">CARTO</a>',
+                subdomains: 'abcd',
+                maxZoom: 19
+            }).addTo(map);
+
+            $.getJSON("../data/duluth_precincts_WGS84.geojson",function(data){
```

Comment:
I'm seeing a couple of errors in the console regarding your map.
One of them is that you never added the jQuery JavaScript library as a resource.

Reply:
Another one is that I don't think you added the GeoJSON data set.